.editorconfig

```diff
@@ -0,0 +1,19 @@
+# Editor configuration, see http://editorconfig.org
```

Comment:
Hmmm this got in here?

Reply:
I think is fine, could go in another PR. Sasha added it https://github.com/cowprotocol/cow-fi/pull/142/commits/ed65b5acb0088c4d150cf926e01ab5237cc2693a